mypyc/test-data/run-valuetype.test

```diff
@@ -0,0 +1,336 @@
+[case testValueTypeBasic]
```

Comment:
Can you also add some irbuild tests for basic functionality to ensure that we generate efficient IR?